### DISCUSSION TITLE
关于 xtls-rprx-vision 这个流控选项的归属疑问

### DISCUSSION THREAD

**Discussion by jvesky** [FranzKafkaYu/x-ui#239] [General]
我看到 X-Ray 官方从 1.72 版本开始推荐使用 xtls-rprx-vision 流控。

但我发现 xtls-rprx-vision 归属在 tls 选项下，如果需要打开 xtls 就会出现下面的这个流程。

1、打开 tls 开关
2、在添加用户下选择 xtls-rprx-vision 流控
3、最后打开 xtls 开关

在打开 xtls 开关后，只有 xtls-rprx-origin 和 xtls-rprx-direct 两个流控。

tls 开关下的选项如图
![image](https://user-images.githubusercontent.com/68280085/212463680-d8536c22-24eb-4ff9-9bc7-b165f160a9d4.png)

xtls 开关下的选项如图
![image](https://user-images.githubusercontent.com/68280085/212463689-7b7fc3b6-44e8-469e-a666-446618a36424.png)

所以这让我有了疑问，是不是把 xtls-rprx-vision 放错了位置？
因为我不懂技术，如果闹出笑话，还请见谅。

感谢您的付出，望解答，谢谢！

**Comment by SpeedPartner** [comment]
就是这样的,tls下的vision流控.
[Xray-core v1.6.2](https://github.com/XTLS/Xray-core/releases/tag/v1.6.2)

**Reply by jvesky** [reply]
谢谢您从百忙之中抽空给我解答此疑惑，非常感谢。
